packages/@react-spectrum/color/src/Gradients.tsx

```diff
@@ -0,0 +1,130 @@
+
+export const generateRGB_R = (orientation, dir, zValue) => {
```

Comment:
Should we add types to these utility methods?
```suggestion
export const generateRGB_R = (orientation:OrientationType, dir:boolean, zValue:number) => {
```